intl/msg_hash_us.h

```diff
@@ -3756,6 +3865,10 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_CONFIG_SAVE_ON_EXIT,
    "Save changes to the configuration file on quit."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_CONFIG_SAVE_ON_EXIT,
+   "Save config to disk on exit. Useful for changes made in the menu. Overwrites the config file, #include's and comments are not preserved."
```

Comment:
I think the first "config" could be expanded in a bit more detail, considering this is a short string. But that's my personal opinion.